staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go

```diff
@@ -997,12 +997,16 @@ func printGlusterfsVolumeSource(glusterfs *corev1.GlusterfsVolumeSource, w Prefi
 }
 
 func printGlusterfsPersistentVolumeSource(glusterfs *corev1.GlusterfsPersistentVolumeSource, w PrefixWriter) {
+	endpointsNamespace := ""
```

Comment:
prefer `<unset>`, see https://github.com/kubernetes/community/blob/master/contributors/devel/sig-cli/kubectl-conventions.md#output-conventions:

> For fields that can be explicitly unset (booleans, integers, structs), the output should say `<unset>`

Reply:
can you also add a unit test for the unset and set cases?

Reply:
done!